deploy/cloudformation/ec2-types.yml

```diff
@@ -0,0 +1,195 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
Can you please add an explanation of how to use it with our current cloudformation script?